mermithid/misc/FakeTritiumDataFunctions.py

```diff
@@ -0,0 +1,655 @@
+'''
+Generate binned or pseudo unbinned data
+Author: T. Weiss, C. Claessens, Y. Sun
+Date:4/6/2020
+'''
+
+from __future__ import absolute_import
+
+import numpy as np
+import math
+
+from scipy.special import gamma
+from scipy import integrate
+from scipy import stats
+from scipy.optimize import fsolve
+from scipy import constants
+from scipy.interpolate import interp1d
+from scipy.signal import convolve
+import random
+import os
+
+from morpho.utilities import morphologging
+logger = morphologging.getLogger(__name__)
+
+"""
+Constants and functions used by processors/TritiumSpectrum/FakeDataGenerator.py
+"""
+
+"""
+Physical constants
+"""
+
+me = 510999. #eV
```

Comment:
Maybe it is worth moving these constants in the https://github.com/project8/mermithid/blob/master/mermithid/misc/Constants.py file and put these as functions; this will allow to avoid duplicated values and potential conflicts